javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/ignore/IgnoredTypesConfigurer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.extension.ignore;
+
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.javaagent.extension.Ordered;
+
+/**
+ * An {@link IgnoredTypesConfigurer} can be used to override built-in instrumentation restrictions:
```

Comment:
"override"? Not "augment" or "change"?

Reply:
Hmm, "augment" sounds much better here 👍